eva_submission/nextflow/accession_and_load.nf

```diff
@@ -410,9 +416,8 @@ process calculate_statistics_vcf {
     pipeline_parameters += " --spring.batch.job.names=calculate-statistics-job"
 
     pipeline_parameters += " --input.vcf.aggregation=" + aggregation.toString().toUpperCase()
-    pipeline_parameters += " --input.vcf=" + vcf_file.toRealPath().toString()
+    pipeline_parameters += " --input.vcf=" + file(vcf_files[0]).toRealPath().toString() // If there are multiple file only use the first
```

Comment:
As far as I can tell, the code is using the file ID rather than the filename, so this is probably okay... I'm actually wondering, since the file ID is the analysis accession, whether we've been computing the same stats multiple times for the same analysis if it contains multiple files...